app/components/EditEntry.js

```diff
@@ -57,22 +91,66 @@ export default class EditEntry extends React.Component {
   };
 
   render() {
-    const { id, entry } = this.state;
+    const { id, entry, showColorPicker: showColorPicker } = this.state;
+
+    const defaultBgColor = '#fff';
+    const defaultColorOptions = [
```

Comment:
This should be extracted outside of the render function I think. Maybe in its own file or at the top of this file.